Config.properties

```diff
@@ -0,0 +1,14 @@
+#Environment Configuration
+VERSION=/v51.0
+SERVICE=/services/data
+LOGIN=https://login.salesforce.com/services/oauth2/token
+loginURL=https://login.salesforce.com/
+
+#Driver configuration
+browser = CHROME
+headless = false
+implicitTime = 20
+explicitTime = 30
+sleepTime = 500
+language = en
+testEnvironment = saul
```

Comment:
testEnvironment = saul is necessary or It can be different value

Reply:
Can be specified in the jenkins environment variables injection as TEST_ENVIRONMENT. I'll check it out, I think it needs to be set in another file.

Reply:
Changed it to dev03